src/client.cpp

```diff
@@ -80,7 +74,7 @@ ADDON_STATUS ADDON_Create(void* hdl, void* props)
   settings.ReadFromAddon();
 
   /* Create connection to NextPVR KODI TV client */
-  g_pvrclient = new cPVRClientNextPVR();
+  g_pvrclient       = new cPVRClientNextPVR();
```

Comment:
Why introduce the whitespace?

Reply:
Accident moving code back and forth from Windows and Ubuntu.  I will fix it but won't  bother with an amendment.